src/GoBoard.d.ts

```diff
@@ -0,0 +1,75 @@
+export type Vertex = [number, number];
+export enum Sign {
+    White = -1,
+    Empty = 0,
+    Black = 1,
+}
+export type SignMap = Sign[][];
+
+declare const alpha: string;
```

Comment:
I'm not familiar with TypeScript definition files. What's the point of declaring a variable if it isn't exported?

Reply:
If you are talking about `alpha`, it fine to remove `declare const alpha: string`, no any impact just I write it incidentally. So I need remove it?

If you are takling about why need export `SignMap`, there is actually no impact if not export it, Typescript can still check the type inside. For example with `Vertex`:
If not export `Vertex`:
```typescript
import Board from '@sabaki/go-board';
const board = Board.fromDimensions(19);
let vertex = [9, '4'];
board.get(vertex); // error here because [9, '4'] is not Vertex aka [number, number]
vertex = [9, '4', 0]; // no error here
```
still can catch error, but if export `Vertex` and use it:
```typescript
import Board, { Vertex } from '@sabaki/go-board';
const board = Board.fromDimensions(19);
let vertex: Vertex = [9, '4']; // error here first
board.get(vertex);
vertex = [9, '4', 0]; // still error here 
```
So, although on both situations can get error, I think when writing code in Typescript, it is better to appoint the type when define a variables that Typescript cannot clearly know its correct type that we want.

Reply:
I'm talking about `alpha`. I think it's better to remove them.

Reply:
Removed in https://github.com/SabakiHQ/go-board/pull/3/commits/9ce2874c6bcce502e957e68a60abb71968a78172